src/einsteinpy/symbolic/predefined/vacuum_solutions.py

```diff
@@ -66,6 +66,43 @@ def Kerr(c=constants.c, sch=symbols("r_s"), a=symbols("a")):
 
 
 def KerrNewman(
-    c=constants.c, G=constants.G, sch=symbols("r_s"), a=symbols("a"), Q=symbols("Q")
+    c=constants.c, G=constants.G, eps_0=constants.eps_0, sch=symbols("r_s"), a=symbols("a"), Q=symbols("Q")
 ):
-    raise NotImplementedError
+    """
+    Kerr-Newman Metric in Boyer Lindquist coordinates.
+
+    Parameters
+    ----------
+    c : ~sympy.core.basic.Basic or int or float
+        Any value to assign to speed of light. Defaults to ``c``.
+    eps0 : ~sympy.core.basic.Basic or int or float
```

Comment:
This is trivial I know :(
```suggestion
    eps_0 : ~sympy.core.basic.Basic or int or float
```

Reply:
Also, you forgot to write docstring for `G`